Tools/peg_generator/scripts/test_pypi_packages.py

```diff
@@ -57,19 +57,8 @@ def find_dirname(package_name: str) -> str:
 def run_tests(dirname: str, tree: int) -> int:
     return test_parse_directory.parse_directory(
         dirname,
-        HERE / ".." / ".." / ".." / "Grammar" / "python.gram",
-        HERE / ".." / ".." / ".." / "Grammar" / "Tokens",
         verbose=False,
-        excluded_files=[
-            "*/failset/*",
-            "*/failset/**",
-            "*/failset/**/*",
-            "*/test2to3/*",
-            "*/test2to3/**/*",
-            "*/bad*",
```

Comment:
Why removing the list of excluded files? If I am not mistaken parsing the CPython source will complain about the encoding for some of these files, no?

Reply:
This is in `test_pypi_packages`, so it doesn't have to do with the CPython source. This misunderstanding was probably the reason I added these in the first place. We shouldn't exclude any files, when testing PyPI packages, right? (Even Python files that match `bad*.py` will probably have valid Python syntax in a PyPI package)

Reply:
> This is in `test_pypi_packages`, so it doesn't have to do with the CPython source. This misunderstanding was probably the reason I added these in the first place. We shouldn't exclude any files, when testing PyPI packages, right?

Oh, right right, I missed the file name and though this was `test_parse_directory` :facepalm: